### PR TITLE
Progressbars: animate .horizontal:not(.pulse)

### PR DESCRIPTION
**Pull request by danrabbit**

### CODE REVIEW

File: src/widgets/_progressbars.scss [hunk 1]
```diff
@@ -32,14 +32,43 @@ progressbar {
...
+                        rgba(white, 0.25) 60%,
+                        rgba(white, 0)
+                    );
+
+                &:dir(ltr) {
+                    animation: progress 2s ease infinite;
+                }
+
+                &:dir(rtl) {
+                    animation: progress-rtl 2s ease infinite;
```
Comment (btkostner) [inline src/widgets/_progressbars.scss:64]: I'm not sure if it's supported, but you might save yourself a bit of code by using `animation-direction: reverse`

https://developer.mozilla.org/en-US/docs/Web/CSS/animation-direction

Comment (danrabbit) [reply src/widgets/_progressbars.scss]: You so smart

**Comment by hanaral** [comment]
Don't wanna be that guy, but lightly shaded diagonal stripes like the old macOS progressbar would be pretty cute:
Screenshot from 2021-01-26 18-18-17

File: src/widgets/_progressbars.scss [hunk 2]
```diff
@@ -32,14 +32,40 @@ progressbar {
...
         trough {
             @extend %outset-background;
         }
 
         progress {
             margin-left: -1px;
             margin-right: -1px;
+
+            &:not(.pulse) {
+                animation: progress 2s ease infinite;
```
Comment (cassidyjames) [inline src/widgets/_progressbars.scss:44]: After playing with a bunch of values, this feels nicer to me. As a bonus, it's the same easing used elsewhere.

```suggestion
                animation: progress 1.5s $easing infinite;
```

File: src/widgets/_progressbars.scss [hunk 3]
```diff
@@ -32,14 +32,40 @@ progressbar {
...
+                        rgba(white, 0)
+                    ),
+                    linear-gradient(
+                        to right,
+                        rgba(white, 0),
+                        rgba(white, 0.25) 60%,
+                        rgba(white, 0)
+                    );
+
+                &:dir(rtl) {
```
Comment (cassidyjames) [inline src/widgets/_progressbars.scss:60]: It turns out GTK automatically handles the animation direction for RTL… but not for the inverted-style progress bars… :facepalm: So this should be:

```suggestion
                &.right {
```

**Review by cassidyjames — CHANGES_REQUESTED** [review]
I like this a lot. Couple of tiny tweaks, then it's good imho.

**Review by cassidyjames — APPROVED** [review]
hawt

**Pull request merged**